scripts/cmd/run_enceladus.cmd

```diff
@@ -382,11 +383,18 @@ IF "%1"=="--set-dra" (
     SHIFT
     GOTO CmdParse
 )
+IF "%1"=="--help" (
```

Comment:
same as expressed above

Moreover, Windows-style help is more customarily `/help` or `/?` (I think), but this gradually ceased to be true with many ported programs anyway.

Reply:
All the other parameters are in `--parameter` form and this is not help on the helper script, but the _SparkJobs.jar` execution.